src/lib/stores/wallet/network.ts

```diff
@@ -395,13 +396,63 @@ export const NETWORK_CONFIG: ValueForEachSupportedChain<Network> = {
       ],
     },
   },
+  [1088]: {
+    chainId: 1088,
+    name: 'metis',
+    label: 'Metis',
+    token: 'METIS',
+    id: '0x440',
+    rpcUrl: 'https://andromeda.metis.io/?owner=1088',
+    icon: Metis,
+    color: '#00D2FF',
+    isTestnet: false,
+    subdomain: 'metis.drips.network',
+    gqlName: SupportedChain.Metis,
+    autoUnwrapPairs: [{ name: 'Filecoin', nativeSymbol: 'FIL', wrappedSymbol: 'WFIL' }],
```

Comment:
Is this right for Metis?

Reply:
no!! bad merge here from the old `metis` branch. Thanks for spotting, fixing 👍